### PR TITLE
Fix missing address area on verified company add confirmation page

### PR DESCRIPTION
**Pull request by faissalMT**
## Description of change

Fixes missing address area on confirmation page when adding a verified company.  
Tests have not been added because enabling this feature flag would break tests for other functionality that is outside of the scope of this PR, a forthcoming PR will fix this.

## Test instructions

When the `address-area-company-search` feature flag is enabled, adding a verified company should now show the address area on the confirmation page.

### Before
image

### After

image

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -28829,18 +28829,21 @@
       }
     },
     "minizlib": {
-      "version": "2.1.2",
-      "resolved": "https://registry.npmjs.org/minizlib/-/minizlib-2.1.2.tgz",
-      "integrity": "sha512-bAxsR8BVfj60DWXHE3u30oHzfl4G7khkSuPW+qvpd7jFRHm7dLxOjUk1EHACJ/hxLY8phGJ0YhYHZo7jil7Qdg==",
+      "version": "1.3.3",
```
Comment (mforner13) [inline package-lock.json:28832]: Looks like this is bringin the version back down - were these changes to package-lock.json accidental?

**Review by mforner13 — COMMENTED** [review]
Just a question around the package-lock.json changes

File: package-lock.json (hunk 1 above)
Comment (faissalMT) [reply package-lock.json:28832]: Yes, this was accidental, I'll clear this up

**Review by mforner13 — APPROVED** [review]
Looks good 👍

**Comment by ian-leggett** [comment]
You can run tests against feature flagged work. Have a look at `src/test/functional/cypress/specs/dashboard-new/feature-flag-spec.js`

**Review by ian-leggett — APPROVED** [review]

**Pull request merged**